x-pack/plugins/infra/public/pages/error.tsx

```diff
@@ -44,13 +45,23 @@ export const ErrorPageBody: React.SFC<{ message: string }> = ({ message }) => {
         <EuiPageHeader>
           <EuiPageHeaderSection>
             <EuiTitle size="m">
-              <h1>Oops!</h1>
+              <h1>
+                <FormattedMessage
+                  id="xpack.infra.homePageError.oopsErrorTitle"
+                  defaultMessage="Oops!"
+                />
+              </h1>
             </EuiTitle>
           </EuiPageHeaderSection>
         </EuiPageHeader>
         <EuiPageContent>
           <EuiCallOut color="danger" title={message} iconType={'alert'}>
-            <p>Please click the back button and try again.</p>
+            <p>
+              <FormattedMessage
+                id="xpack.infra.homePageError.clickBackButtonErrorTitle"
```

Comment:
why error title? why  error? => tryAgainDescription or something like this. @maryia-lapata